apps/studio/data/views/view-query.ts

```diff
@@ -18,30 +19,28 @@ export async function getView(
   { id, projectRef, connectionString }: ViewVariables,
   signal?: AbortSignal
 ) {
-  if (!projectRef) {
-    throw new Error('projectRef is required')
-  }
-  if (!id) {
-    throw new Error('id is required')
-  }
+  if (!projectRef) throw new Error('projectRef is required')
+  if (!id) throw new Error('id is required')
 
   let headers = new Headers()
   if (connectionString) headers.set('x-connection-encrypted', connectionString)
 
-  const response = (await get(`${API_URL}/pg-meta/${projectRef}/views?id=${id}`, {
-    headers: Object.fromEntries(headers),
+  const { data, error } = await get('/platform/pg-meta/{ref}/views', {
+    params: {
+      header: { 'x-connection-encrypted': connectionString! },
+      path: { ref: projectRef },
+      query: { id: `${id}` } as any,
```

Comment:
I think it'd be stringified by the fetcher anyway, but dont bother changing it :p